.pipelines/ci.yml

```diff
@@ -1,20 +1,24 @@
 # Azure DevOps Pipeline running CI
 trigger:
-- master
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
```

Comment:
Triggers pipelines first time when new PR is opened to the `master` branch. Other branches does not trigger CI run. It runs CI every time when changes are made to the opened PR.

Changes the default from PR to any branch. 

Documentation for pr triggers: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pr-trigger